### PR TITLE
Typescript! ESLint, scripts, github workflows, more formatting, handlers and more

### PR DESCRIPTION
**Pull request by Brian3647**
Everything added without issues - see #6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,44 @@
 {
 	"name": "shofy",
-	"version": "1.0.0",
+	"version": "0.0.0",
```
Comment (Brian3647) [inline package.json:3]: Do you want me to change this?

Comment (jc-discdev) [reply package.json:3]: I don’t think so, it’s on v1.

**Comment by jc-discdev** [comment]
Nice job.

**Review by jc-discdev — APPROVED** [review]
Perfect work and good made, thanks for your collaboration.

**Pull request merged**